hedgehog-test-laws/hedgehog-test-laws.cabal

```diff
@@ -54,13 +54,12 @@ test-suite test
   build-depends:
       hedgehog
     , base                            >= 3          && < 5
-      -- https://github.com/conal/checkers/issues/44
-    , checkers                        >= 0.5        && < 0.5.5
+    , checkers                        >= 0.5        && < 0.8
```

Comment:
The checkers issues has been fixed and I assume the new version on Hackage has the fix.

Reply:
Latest version on hackage is `0.6` so `< 0.8` will keep CI happy for a while.